modules/cnf-installing-numa-resources-operator.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_content-type: PROCEDURE
+[id="installing-the-numa-resources-operator_{context}"]
+= Installing the NUMA Resources Operator
+
+NUMA Resources Operator deploys resources that allow you to schedule NUMA-aware workloads and deployments. You can install the NUMA Resources Operator using the {product-title} CLI or the web console.
+
+[id="install-operator-cli_{context}"]
+== Installing the NUMA Resources Operator using the CLI
+
+As a cluster administrator, you can install the Operator using the CLI.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a namespace for the NUMA Resources Operator:
+
+.. Save the following YAML in the `nro-namespace.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-numaresources
+----
+
+.. Create the `Namespace` CR:
++
+[source,terminal]
+----
+$ oc create -f nro-namespace.yaml
+----
+
+. Create the operator group for the NUMA Resources Operator:
+
+.. Save the following YAML in the `nro-operatorgroup.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-numaresources-operator
+  namespace: openshift-numaresources
+spec:
+  targetNamespaces:
+  - openshift-numaresources
+----
+
+.. Create the `OperatorGroup` CR:
++
+[source,terminal]
+----
+$ oc create -f nro-operatorgroup.yaml
+----
+
+. Create the subscription for the NUMA Resources Operator:
+
+.. Save the following YAML in the `nro-sub.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-numaresources-operator
+  namespace: openshift-numaresources
+spec:
+  channel: '4.10'
+  name: openshift-numaresources-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+----
+
+.. Create the `Subscription` CR:
++
+[source,terminal]
+----
+$ oc create -f nro-sub.yaml
+----
```

Comment:
Could we add a check in here to show NUMA Resource Scheduler is installed with output ?
_oc get pods -n  openshift-numaresources_

Reply:
This is done in the `Deploying the NUMA-aware secondary pod scheduler` section.